qna/general-posoperator-repeat-invitation-en.md

```diff
@@ -0,0 +1,33 @@
+## Question
+
+How can the assignment from PosDealer to PosOperator be established, if the e-mail-invitation from ft.Portal does not arrive at the PosOperator?
+
+## Metadata tags
+
+lang-en, market-all, portal, PosDealer, PosOperator
+
+## Answer
+---
+slug: /portal-manual/general/posoperator
+title: PosOperator
+---
+
+### Repeat invitation of a PosOperator by the PosDealer
+
```

Comment:
Before resending any invitation please verify the correctness of the email used in the invitation.